Branch1

xJungKim opened the pull request:
Rendered the list of expenses from a static hard-coded list to a user-inputted list. 
Dropdown filter to show expenses per year is now functional.
Implemented a statement to inform users that there is no expenses found within a given year they have selected in the dropdown, if none exist.
Made Expenses.js leaner by creating ExpensesList.js
Added a button to open the form to launch the user input for new expenses with the tag of "Add New Expenses" and added a "Cancel" button to revert back and close the form.
Added a chart to serve as a visualization that allows users to see how much they spent per month. (I'm having trouble getting the visualization to actually show, it's just popping up as blank bars. Could you look into my code to see where I went wrong?)

xJungKim committed:
test

pull req test

xJungKim committed:
Update App.js

xJungKim committed:
7-8-2022

Rendered the list of expenses from a static hard-coded list to a user-inputted list.
Dropdown filter to show expenses per year is now functional.
Implemented a statement to inform users that there is no expenses found within a given year they have selected in the dropdown, if none exist.
Made Expenses.js leaner by creating ExpensesList.js
Added a button to open the form to launch the user input for new expenses with the tag of "Add New Expenses" and added a "Cancel" button to revert back and close the form.
Added a chart to serve as a visualization that allows users to see how much they spent per month.

xJungKim committed:
7-8-2022 (2)

Fixed an error within ChartBar.js that caused the bars to not show up visually as intended.

espaulding commented on src/App.js:
```diff
@@ -1,33 +1,37 @@
 import NewExpense from "./Components/NewExpense/NewExpense";
 import Expenses from "./Components/Expenses/Expenses";
 
+const dummy_expenses = [
```
rename to DUMMY_EXPENSES or if making if not a constant dummyExpenses.

espaulding reviewed (approved):
looking good

xJungKim committed:
7-8-2022 (3)

Changed the color scheme of the chart to match the rest of the website.
Edited the static items to allow the user to not deal with items that already exist.
Made the starting date 2022 to better reflect the present, also allowing users to immediately see results as we would generally be putting in expenses for the current year.
